WordPress/src/main/res/layout/comment_detail_fragment.xml

```diff
@@ -4,17 +4,56 @@
     comment detail displayed from both the notification list and the comment list
 -->
 
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:background="?attr/colorSurface">
 
+    <androidx.cardview.widget.CardView
+        android:id="@+id/header_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginHorizontal="@dimen/margin_extra_large"
+        android:foreground="?attr/selectableItemBackground"
+        app:cardBackgroundColor="@color/background_like_header"
+        app:cardCornerRadius="@dimen/notifications_like_header_radius"
+        app:cardElevation="0dp">
+
+        <!-- This header only display the title from posts -->
+
+        <TextView
+            android:id="@+id/text_post_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_extra_large"
+            android:layout_marginEnd="@dimen/margin_extra_extra_medium_large"
+            android:ellipsize="end"
+            android:lines="1"
+            android:paddingVertical="@dimen/page_list_row_vertical_padding"
+            android:textColor="?attr/colorOnSurface"
+            android:textSize="@dimen/text_sz_medium"
+            tools:text="Snippet" />
+
+        <ImageView
+            android:id="@+id/header_arrow"
+            android:layout_width="@dimen/notifications_like_header_icon_size"
+            android:layout_height="@dimen/notifications_like_header_icon_size"
+            android:layout_marginEnd="@dimen/margin_large"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_arrow_right_white_24dp"
```

Comment:
I noticed the Figma design contains a brand-new icon for the arrow.
<img width="400" alt="Screenshot 2024-05-03 at 12 08 43" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/48921582-db0e-4e56-b7f7-f7357bafdc0c">

Reply:
Nice catch!